conftest.py

```diff
@@ -1,27 +1,12 @@
 # pylint: disable=wrong-import-position,redefined-outer-name,unused-wildcard-import,wildcard-import
-import gevent  # isort:skip # noqa
-from gevent import monkey, Timeout  # isort:skip # noqa
+# type: ignore
+from gevent import monkey
 
-monkey.patch_all(subprocess=False, thread=False)  # isort:skip # noqa
+monkey.patch_all(subprocess=False, thread=False)
 
-import asyncio
-import contextlib
-import datetime
-import os
-import signal
-import subprocess
-import sys
-import time
+# isort:split
```

Comment:
That's cool! I didn't know about that one.

Reply:
Yep, it's new in isort >= 5.

Very useful since it now allows multiple sections to be sorted separately.